utils/text/transformers.py

```diff
@@ -180,4 +184,12 @@ def snake_case_to_lower_camel_case(string):
     """Convert `snake_case` string to `camelCase`"""
     camel_string = snake_case_to_camel_case(string)
     camel_string = camel_string[0].lower() + camel_string[1:]
-    return camel_string
\ No newline at end of file
+    return camel_string
+
+
+def pascal_case_to_snake_case(string):
```

Comment:
Use the variable name `s` instead of `string`

Reply:
Done